modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} Ingress Controller for dual-stack support
+
+When you install a {product-title} cluster on a IPv4/IPv6 dual-stack enabled bare metal infrastructure, your cluster to is externally reachable by {product-title} routes.
+
+The Ingress Controller automatically serves services that have both IPv4 and IPv6 endpoints, but you can configure the Ingress Controller for single-stack or dual-stack services.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on bare metal.
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+To have the Ingress Controller serve traffic over IPv4/IPv6 to a workload, you can create or modify a service by setting the `ipFamilies` and `ipFamilyPolicy` fields.
+
+. Create service by entering the following command:
++
+[source,terminal]
+----
+$ oc get service <service_name>
+----
++
+. Check your services by entering the following command:
++
+[source,terminal]
+----
+$ oc get service <service_name> -o yaml
+----
++
+. Edit the `ipFamilies` and `ipFamilyPolicy` fields in the service YAML file:
++
+.Sample Network YAML file
+[source,yaml]
+----
+apiVersion: v1
+  kind: Service
+  metadata:
+    creationTimestamp: yyyy-mm-ddT00:00:00Z
+    labels:
+      name: <service_name>
+      manager: kubectl-create
+      operation: Update
+      time: yyyy-mm-ddT00:00:00Z
+    name: <service_name>
+    namespace: <namespace_name>
+    resourceVersion: "<resource_version_number>"
+    selfLink: "/api/v1/namespaces/<namespace_name>/services/<service_name>"
+    uid: <uid_number>
+  spec:
+    clusterIP: 172.30.0.0/16
+    clusterIPs: <3>
```

Comment:
Customers can manage if the sequence is 3, 1, 2, but it might still be OK if you revise to 1, 2, and 3 and reorder the callout list.